protocol/auditlog.go

```diff
@@ -11,10 +11,21 @@ import (
 )
 
 type directoryHistory struct {
+	*AudState
 	addr      string
-	signKey   sign.PublicKey
-	snapshots map[uint64]*DirSTR
-	latestSTR *DirSTR
+	snapshots []*DirSTR
+}
+
+// caller validates that initSTR is for epoch 0
+func newDirectoryHistory(addr string, signKey sign.PublicKey, initSTR *DirSTR) *directoryHistory {
+	a := NewAuditor(signKey, initSTR)
```

Comment:
Please use either `a` or `aud` for the consistency. You used `aud` in https://github.com/coniks-sys/coniks-go/pull/170/files?diff=unified#diff-8eead7b3ac1a29e7ad9e067fb3d9aaf8R46